lang/ja.json

```diff
@@ -207,22 +207,40 @@
     "GMTOOLKIT.Macro.AddXP": "経験値獲得",
     "GMTOOLKIT.Macro.ResetFortune": "幸運点初期化",
     "GMTOOLKIT.Macro.MakeSecretPartyTest": "秘密のパーティテスト",
-    "GMTOOLKIT.Macro.SendDarkWhispers": "「闇の囁き」送信",
+    "GMTOOLKIT.Macro.SendDarkWhispers": "「暗黒のささやき」送信",
     "GMTOOLKIT.Macro.ToggleSceneVisibilityandLight": "シーンの視界と光源の切替",
     "GMTOOLKIT.Macro.SetTokenVisionandLight": "トークンの視界と光源を設定",
     "GMTOOLKIT.Macro.PullEveryonetoScene": "全員をシーンに引き寄せる",
     "GMTOOLKIT.Macro.Simplyd100": "100面ダイス",
     "GMTOOLKIT.Macro.GMToolbox": "GM Toolbox",
     "GMTOOLKIT.Macro.ToggleCompendiumPackVisibility": "Toggle Compendium Pack Visibility",
     
-    "GMTOOLKIT.Table.DarkWhispers": "Dark Whispers",
-
-    "GMTOOLKIT.Dialog.Maintenance.Hint": "Remove all macros or tables in the 'GM Toolkit' Macro directory or RollTable folder, if that folder exists, and import the latest compendium versions.<br><br>Any customisations will be lost. Move these out of the GM Toolkit directory first if you want to preserve any changes you have made.",
-    "GMTOOLKIT.Dialog.Maintenance.Macros": "Macros",
-    "GMTOOLKIT.Dialog.Maintenance.Tables": "Tables",
-    "GMTOOLKIT.Settings.Maintenance.menu.name" : "Module Maintenance",
-    "GMTOOLKIT.Settings.Maintenance.menu.hint" : "Replace imported GM Toolkit macros and/or tables with latest compendium versions.", 
-    "GMTOOLKIT.Settings.Maintenance.menu.label" : "Update GM Toolkit Content",
+    "GMTOOLKIT.Table.DarkWhispers": "暗黒のささやき",
+
+    "GMTOOLKIT.Dialog.Maintenance.Hint": "「GM Toolkit」マクロディレクトリやロール表フォルダの全てのマクロとロール表を削除し、最新の事典版をインポートする。<br><br>カスタマイズした内容は全て失われます。残しておく必要がある場合、先にGM Toolkitフォルダからそれらを移動してください。",
+    "GMTOOLKIT.Dialog.Maintenance.Macros": "マクロ",
+    "GMTOOLKIT.Dialog.Maintenance.Tables": "テーブル",
+    "GMTOOLKIT.Settings.Maintenance.menu.name" : "モジュール管理",
+    "GMTOOLKIT.Settings.Maintenance.menu.hint" : "インポートした「GM Toolkit」のマクロやロール表を、事典の最新版に置き換える。", 
+    "GMTOOLKIT.Settings.Maintenance.menu.label" : "GM Toolkitのコンテンツ更新",
+
+    "Climb":"登攀",
+    "Consume Alcohol": "大酒飲み",
+    "Drive":"操縦",
+    "Ride":"騎乗",
+    "Dodge":"回避",
+    "Athletics":"運動",
+    "Stealth":"隠密",
+    "Track":"追跡",
+    "Cool":"冷静さ",
+    "Endurance":"肉体抵抗",
+    "Charm Animal":"てなずけ",
+    "Gamble":"賭博",
+    "Charm":"話術",
+    "Gossip":"世間話",
+    "Pray":"祈念",
+    "Intuition":"威圧",
+    "Perception":"知覚",

```

Comment:
I'll update the Hud to use available system keys, rather than duplicate them in Toolkit. 
Reference: https://github.com/moo-man/WFRP4e-FoundryVTT/blob/a5d52c49cc54a215b61e306259de6345d63bad0d/lang/en.json#L967-L1003

Reply:
I've also submitted a [PR](https://github.com/moo-man/WFRP4e-FoundryVTT/pull/1156) for the rest of these to be added to the base system, so they should all be redundant in the Toolkit translation files.